js/interface.js

```diff
@@ -2276,12 +2287,19 @@ function checkSubmissionStatus(origin, iosSubmissions) {
 }
 
 function submissionChecker(submissions) {
+  debugger;
+
   var asub = _.filter(submissions, function(submission) {
     return submission.data.submissionType === "appStore" && submission.platform === "ios";
   });
 
   var completedAsub = _.filter(asub, function(submission) {
-    return submission.data.status === "completed";
+    return submission.status === "completed";
+  });
+
+  //Get the Submission data from the first completed submission, it has the certification values that are in use on the app store.
```

Comment:
why the first one? previous should be current-1, right?

Reply:
As the comment says @squallstar "Get the Submission data from the first completed submission, it has the certification values that are in use on the app store." This will prevent us from fetching wrong p12 data that was obtained from a submission done with another p12 besides the first one.  It is more of an issue for the Enterprise submission that it is for the App Store, so Ikept as it is for both.